src/styles/blocks/section.scss

```diff
@@ -0,0 +1,70 @@
+.section {
```

Comment:
every section have asme paddings and number of grid columns, you can create class sections with paddings and grid and modify only if ot's different using modifier